### ISSUE TITLE
WCAG 4.1.2: Ensures all ARIA attributes have valid values

### ISSUE THREAD

**Issue by kattepalli** [microsoft/BotFramework-WebChat#3889]
**Issue**
Ensures all ARIA attributes have valid values (aria-valid-attr-value)

**Target application**
Microsoft Virtual Agent

**Element path**
.webchat--css-fnyqe-1au3coz.webchat__basic-transcript__activity[aria-label="activity"]:nth-child(2) > .webchat__stacked-layout.webchat--css-fnyqe-rctxl4.webchat--css-fnyqe-opzqsn

**Snippet**
div aria-labelledby="webchat__stacked-layout__id--5agnk" aria-roledescription="activity" class="webchat__stacked-layout webchat--css-fnyqe-rctxl4 webchat--css-fnyqe-opzqsn webchat__stacked-layout--no-message webchat__stacked-layout--top-callout" role="...

**How to fix**
Fix all of the following:
- Invalid ARIA attribute value: aria-labelledby="webchat__stacked-layout__id--5agnk"

**Repro Steps:**
Launch URL: https://vsa.services.microsoft.com/en-us/v1.0/configurable.html 
Enter Partner id as "7d74cf73-5217-4008-833f-87a1a278f2cb" and select submit button.
Navigate all the controls of "Virtual Agent" section.
Verify whether the above steps 3 are accessible and MAS compliant or not.

**Actual results:**
ARIA attributes does not have valid values.

**Expected Result:**
Ensures all ARIA attributes have valid values.

**MAS Reference:**
[MAS 4.1.2 - Name, Role, Value](https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7b248054a6-5e68-4771-9e1e-242fb5025730%7d)

**Test Environment :**
OS: Win 10 20H2
Build: OS Build 19042.804
Platform: Version 90.0.803.0 (Official build) dev (64-bit)
Screen Reader: Narrator, NVDA, JAWS
#URL: Skylight Virtual Agent
<img width="960" alt="MAS4 1 2_Ensures all ARIA attributes have valid values" src="https://user-images.githubusercontent.com/30458420/117964287-0e59a680-b33f-11eb-8a19-d96d662f9e53.png">

**Comment by tdurnford** [comment]
@corinagum Can you take a look at this?

**Comment by corinagum** [comment]
Please update to the latest version of Web Chat (4.13.x) and verify the bug. These issues should be addressed.

**Comment by corinagum** [comment]
Closing due to inactivity. @kattepalli if you are able to test this on the latest version of Web Chat and still encounter these errors, leave a comment here and I can reopen the issues. Thanks.
